CodeMaid/UI/Dialogs/Options/Cleaning/CleaningFileTypesDataTemplate.xaml

```diff
@@ -1,6 +1,7 @@
 <ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:local="clr-namespace:SteveCadwallader.CodeMaid.UI.Dialogs.Options.Cleaning"
+        xmlns:properties="clr-namespace:SteveCadwallader.CodeMaid.Properties"
```

Comment:
In other files the convention is `xmlns:p` so it would be good to stick to that here too.

Reply:
'properties' are the names that Tom-englert/resxresourcemanager use by default, so they are still for faster translation.

Reply:
I understand a lot of the code was auto-generated by a tool, but we've been really diligent about trying to have as clean of a code base as possible.  To accept the pull request the auto-generated code will have to be updated to match conventions of the existing code base.

Reply:
has been modified ！
91af8fdb442cd0bae3bd6ce5ed563fd5d394c227